src/CodeGenerator/AutoloadGenerator.php

```diff
@@ -7,87 +7,93 @@
 
 namespace Zend\Di\CodeGenerator;
 
-use Zend\Code\Generator\DocBlockGenerator;
-use Zend\Code\Generator\FileGenerator;
-use Zend\Code\Generator\ClassGenerator;
-use Zend\Code\Generator\MethodGenerator;
-use Zend\Code\Generator\PropertyGenerator;
-use Zend\Code\Generator\PropertyValueGenerator;
+use SplFileObject;
+use Throwable;
+use Zend\Di\Exception\GenerateCodeException;
+use function array_keys;
+use function array_map;
+use function file_get_contents;
+use function implode;
+use function str_repeat;
+use function strtr;
+use function var_export;
 
 class AutoloadGenerator
 {
     use GeneratorTrait;
 
-    private $namespace;
+    const CLASS_TEMPLATE = __DIR__ . '/../../templates/autoloader-class.template';
+    const FILE_TEMPLATE = __DIR__ . '/../../templates/autoloader-file.template';
 
     /**
-     * @param string $namespace
+     * @var string
      */
+    private $namespace;
+
     public function __construct(string $namespace)
     {
         $this->namespace = $namespace;
     }
 
-    private function generateAutoloaderClass(array &$classmap)
+    private function writeFile(string $filename, string $code): void
```

Comment:
Only a short notice: space before colon is missing.

Reply:
Strange, why didn't the cs-check complain about it? Maybe an outdated version. Thanks for pointing this out. 👍

Reply:
@froschdesign I ran `composer update`. The locked version of zend-coding-standard was not out of date. cs-check is still not complaining. Worth a bug report on [zendframework/zend-coding-standard]? For the sake of consistency I'd argue to leave it this way for the moment and fix this in a separate pull request. Is this OK for you?